docs/source/checks/nlp/data_integrity/plot_unknown_tokens.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+
+.. _nlp__unknown_tokens:
+
+Unknown Tokens
+**************
+
+This notebook provides an overview for using and understanding the Unknown Tokens check.
+
+**Structure:**
+
+* `What is the purpose of the check? <#what-is-the-purpose-of-the-check>`__
+* `Generate data & model <#generate-data-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is the purpose of the check?
+==================================
+
+The Unknown Tokens check is designed to help you identify samples that contain tokens unsupported by your tokenizer.
+These unsupported tokens can lead to poor model performance, as the model may not be able to understand the meaning
```

Comment:
unsupported (*2) -> not supported

(I think "unsupported" doesn't exist as a word in that context. It is a valid word but with a bit of a different meaning)